modules/users/users.go

```diff
@@ -18,6 +18,7 @@ type User struct {
 	Password 		string `json:"password"`
 	Firstname   	string `json:"firstname"`
 	LastName 		string `json:"lastname"`
+	IsAdmin 		bool 	`json:"isAdmin"`
```

Comment:
for this isAdmin field, we dont want to be passing this around with the user (we shouldn't expect them to tell us their role in any request). We can hide fields using `json:"-"` I believe. Roles should only be passed using a signed JWT with a client.

Reply:
But do we want the isAdmin during the post request, when we make a user. Otherwise how will we initialize isAdmin, when creating a new user?

Reply:
we set it to a normal prof only. Ideally if we want an admin account we need to manually give it privileges for an added layer of security. If this becomes a pain we can look into a protected endpoint to elevate a users role in a future sprint. Having a public API endpoint where someone can make a user and give it admin roles seems pretty sketchy to me.

Reply:
Makes sense. Let me add these changes.

Reply:
@jakecomp @jberryman566  I have added the suggestions. For now, I have taken out the JWT logic, as that's a separate ticket for now. I have left a comment and I am currently returning a dummy JWT. This can be replaced by the actual JWT in the next ticket that @jakecomp has. Lol, sorry for the confusion guys